go/processor/floria/processor_test.go

```diff
@@ -504,3 +509,102 @@ func TestProcessor_AccessListIsNotCreatedIfTransactionHasNone(t *testing.T) {
 
 	setUpAccessList(transaction, context, tosca.R09_Berlin)
 }
+
+func TestProcessor_blobCheckReturnsErrors(t *testing.T) {
+	tests := map[string]struct {
+		transaction tosca.Transaction
+		blockParams tosca.BlockParameters
+		expectError bool
```

Comment:
nit: to make sure that tests fail for the expected reasons, you should check that the error message contains some expected issue report